core/src/Microsoft.Identity.Core.csproj

```diff
@@ -99,6 +99,10 @@ in ADAL and MSAL and to ease unit testing. No projects should take a dependency
     <Compile Include="Platforms\iOS\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Telemetry\XmsCliTelemInfoParser.cs" />
```

Comment:
You can thank VS, but you should delete this itemgroup  chunk here.

Reply:
done